CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,6 @@
+- Fix logs (Other Error and forwarding related traces) (issue #1794)
```

Comment:
```suggestion
- Add: new alarm (id number 6) for forwarding errors (#1794)
- Fix: use "Forwarding Error" instead of "Other Error" in WARN traces (#1794)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3749/commits/49298094b6110acc3cb09e5c5a23442fa93ab5f7